### DISCUSSION TITLE
Alternate web server solution for the demo project

### DISCUSSION THREAD

**Discussion by dafvid** [nuejs/nue#38] [Show and tell]
I replaced the server.js with node module http-server.

My package.now looks like this
```
...
"scripts": {
    "start": "bun minify && bun compile && http-server www -p 8000 -d false",
    "serve": "cd www && ../scripts/server.js",
    "compile": "./scripts/compile.js",
    "minify": "./scripts/minify.js",
    "render": "./scripts/render.js"
  },
...
```

this replaces the serve script

**Comment by tipiirai** [comment]
I was hoping to find a native solution in Node like this. But unfortunately I get `sh: http-server: command not found` when running the command

**Reply by dafvid** [reply]
yea it's a module unfortunately. [http-server](https://github.com/http-party/http-server). Your existing solution seems to be the easiest one without any dependencies.

**Reply by mannix-lei** [reply]
try to use `"start": "bun minify && bun compile && npx http-server www -p 8000 -d false"`, it will work.

**Comment by tipiirai** [comment]
I'm hesitant to add http-server as a dependency since it's quite a massive package (8.5M) with many dependencies:

https://github.com/http-party/http-server/blob/master/package.json#L87
